app/controllers/profile_controller.rb

```diff
@@ -11,4 +11,45 @@ def index
     @stories = Kaminari.paginate_array(get_stories(user))
                        .page(params[:page])
   end
+
+  def ban_user
+    ban_or_remove(true)
+  end
+
+  def remove_ban
+    ban_or_remove(false)
+  end
+
+  private
+
+  def ban_or_remove_mailer(user, banned)
+    return unless user.any?
+
+    if banned
+      BannedMailer.add_ban_email(user.first).deliver_now
+    else
+      BannedMailer.remove_ban_email(user.first).deliver_now
+    end
+  end
+
+  def ban_or_remove(banned)
```

Comment:
I wonder if there's a more expressive name we can give this method. The two wrappers above (`ban_user` and `remove_ban`) make this very clear, but as a standalone method, I had to do a double take. Calling `ban_or_remove(false)` is a little confusing, but that could also just be me.

Would something like `ban` be too generic?